docs/api.rst

```diff
@@ -4748,6 +4832,67 @@ AuditLogDiff
 
         :type: Optional[:class:`ThreadSortOrder`]
 
+    .. attribute:: title
+
+        The title of an onboarding prompt or an onboarding prompt option being changed.
+
+        :type: :class:`str`
+
+    .. attribute:: options
+
+        The list of options of an onboarding prompt being changed.
+
+        :type: List[:class:`OnboardingPromptOption`]
+
+    .. attribute:: single_select
+
+        The onboarding prompt is single select or not.
+
+        :type: :class:`bool`
+
+    .. attribute:: required
+
+        The onboarding prompt option is required or not.
+
+        :type: :class:`bool`
+
+    .. attribute:: in_onboarding
+
+        The onboarding prompt option is in onboarding or not.
+
+        :type: :class:`bool`
+
+    .. attribute:: default_channel
+
+        The list of default channels of an onboarding being changed.
+
+        :type: List[:class:`abc.GuildChannel`]
+
+    .. attribute:: enable_default_channels
+
+        The onboarding shows default channels or not.
+
+        :type: :class:`bool`
+
+    .. attribute:: enable_onboarding_prompts
+
+        The onboarding shows onboarding prompts or not.
+
+        :type: :class:`bool`
+
+    .. attribute:: prompts
+
+        The list of prompts of an onboarding being changed.
+
+        :type: List[:class:`OnboardingPrompt`]
+
+    .. attribute:: default_channels
+
+        The list of default channels of an onboarding being changed.
+
+        :type: List[:class:`~.abc.GuildChannel`]
```

Comment:
```suggestion
    .. attribute:: prompts

        The list of onboarding prompts being changed.

        :type: List[:class:`OnboardingPrompt`]

    .. attribute:: default_channels

        The list of default channels for onboarding being changed.

        :type: List[:class:`~.abc.GuildChannel`]
```

Reply:
closing since audit logs were not added in api-docs pr. 2b81f4c90af51506ae7f6c0843f514b58be80f7a